pkg/serverless/trace/trace.go

```diff
@@ -126,6 +142,20 @@ func StartServerlessTraceAgent(enabled bool, loadConfig Load, lambdaSpanChan cha
 	return noopTraceAgent{}
 }
 
+func startTraceAgentConfigEndpoint(rcService *remoteconfig.CoreAgentService, tc *config.AgentConfig) {
+	if pkgconfigsetup.IsRemoteConfigEnabled(pkgconfigsetup.Datadog()) && rcService != nil {
+		statsdNoopClient := &statsd.NoOpClient{}
+		api.AttachEndpoint(api.Endpoint{
+			Pattern: "/v0.7/config",
+			Handler: func(r *api.HTTPReceiver) http.Handler {
+				return remotecfg.ConfigHandler(r, rcService, tc, statsdNoopClient, timing.New(statsdNoopClient))
+			},
+		})
+	} else {
+		log.Debug("Could not start trace agent config endpoint")
```

Comment:
```suggestion
		log.Debug("Not starting trace agent config endpoint")
```

Just cuz we don't want people thinking that if they see this in their debug logs that something went wrong.

Reply:
Good call, updated